ApplyToBecomeInternal/ApplyToBecomeInternal/TagHelpers/SummaryListRowTagHelper.cs

```diff
@@ -33,6 +33,9 @@ public class SummaryListRowTagHelper : InputTagHelperBase
 		[HtmlAttributeName("value-width")]
 		public string ValueWidth { get; set; }
 
+		[HtmlAttributeName("highlight-negative-value")]
+		public string HighlightNegativeValue { get; set; }

```

Comment:
You can make HighlightNegativeValue a bool and therefore remove the parsing bit of line 60

Reply:
I have made the change, but this will now require exact c# string literals for 'true/false'. Empty or anything else will lead to compile time errors but I believe this is acceptable behaviour.